core/src/main/java/org/apache/gravitino/storage/relational/service/TagMetaService.java

```diff
@@ -55,6 +58,18 @@ public static TagMetaService getInstance() {
     return INSTANCE;
   }
 
+  private static final Map<MetadataObject.Type, Function<List<Long>, Map<Long, String>>>
```

Comment:
Shall  RoleService  replace the code with similar logic?

Reply:
Yes,  the code in `RoleService` uses a temporary value in a method to solve it. They are mostly the same and have minor differences. 

Do you mean we may need to try to merge them?

Reply:
Yes, we should merge them. I guess

Reply:
fix